daemons/attrd/attrd_elections.c

```diff
@@ -58,7 +58,7 @@ attrd_start_election_if_needed(void)
 bool
 attrd_election_won(void)
 {
-    return (election_state(writer) == election_won);
+    return (stand_alone || election_state(writer) == election_won);
```

Comment:
If we have a single-node cluster, even this might work without change

Reply:
Yep, I can revert this and the part that skips attrd_election_init and everything works fine.